changehc/delphi_changehc/update_sensor.py

```diff
@@ -158,10 +156,10 @@ def geo_reindex(self, data):
                                                  mega_col=geo)
         elif geo == "state":
             data_frame = gmpr.replace_geocode(data, "fips", "state_id", new_col="state")
-        elif geo == "msa":
-            data_frame = gmpr.replace_geocode(data, "fips", "msa")
-        elif geo == "hrr":
-            data_frame = gmpr.replace_geocode(data, "fips", "hrr")
+        elif geo in ["msa", "hrr", "hhs"]:
+            data_frame = gmpr.replace_geocode(data, "fips", geo)
+        else:
+            data_frame = gmpr.replace_geocode(data, "fips", "nation")
```

Comment:
```suggestion
        else:
            data_frame = gmpr.replace_geocode(data, "fips", geo)
```
Any reason to not include "nation" in the catch-all?

Reply:
Nope, good catch.